2009-flu-uk.qmd

```diff
@@ -1 +1,437 @@
-# 2009 pandemic in the UK
\ No newline at end of file
+# 2009 pandemic in the UK
+
+```{r}
+#| include: false
+source("common.R")
+```
+
+```{r, echo=FALSE}
+r_output <- function(path, highlight = NULL) {
+  if (is.null(highlight)) {
+    prefix <- "```r"
+  } else {
+    prefix <- sprintf('```{.r code-line-numbers="%s"}', highlight)
+  }
+  writeLines(c(prefix, readLines(path), "```"))
+}
+set.seed(1) # always the same
+knitr::knit_hooks$set(small_margins = function(before, options, envir) {
+  if (before) {
+    par(mar = c(4, 4, .1, .1))
+  }
+})
+```
+## Introduction
+
+The 2009 A/H1N1 influenza pandemic posed a significant public health challenge in England and Wales, requiring rapid and evidence-based decision-making to mitigate its impact. This chapter explores the use of `odin` and `monty` to model the dynamics of the 2009 A/H1N1 influenza pandemic in England and Wales incorporating changes in social contacts during holiday periods. We demonstrate how to construct, fit, and analyse a compartmental model to infer key epidemiological parameters.
```

Comment:
Do you want to add a link to a paper or a webpage or similar?

Reply:
I have added two references to papers from the UK 2009 pandemic